zstacklib/zstacklib/utils/lv_metadata.py

```
@@ -255,6 +255,43 @@ def _do_cleanup(self, metadataPath):
+            logger.warn("cleanup_all: vgUuid missing on cmd")
+            return {'cleanedCount': 0, 'failedCount': 0}
+        if not self._SAFE_VG_RE.match(vg_uuid):
+            logger.warn("cleanup_all: invalid vgUuid: %r", vg_uuid)
+            return {'cleanedCount': 0, 'failedCount': 0}
+
+        lvm = self._lvm
+        cleaned = 0
+        failed = 0
+        try:
+            metadata_lvs = scan_metadata_lvs(vg_uuid, lvm.list_lvs)
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)